tag/map_codec.go

```diff
@@ -195,10 +195,7 @@ func Decode(bytes []byte) (*Map, error) {
 	for !eg.readEnded() {
 		typ := keyType(eg.readByte())
```

Comment:
This is dead code now. We won't have any other types than string. I will do a round of internal cleanup to rewrite this section.

Reply:
Great, ping me if you think I might be useful somewhere.